src/InputWithLabel.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function InputWithLabel(props) {
```

Comment:
Let's be consistent with our coding pattern. Here you are not destructuring your prop object while in other components you are. It's up to you which you choose, just be sure to be consistent across your code base :)